reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -104,8 +103,11 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
             val tableRows = allIds.map { id ->
                 val scanResult = scanRecord?.scanResults?.find { it.id == id }
 
-                val scopes = project.scopes.filter { id in it }.let { scopes ->
-                    excludes.scopeExcludesByName(scopes).toSortedMap()
+                val scopes = project
+                    .scopes
+                    .filter { id in it }
+                    .map { it.name }
+                    .let { scopesNames -> excludes.scopeExcludesByName(scopesNames).toSortedMap()
```

Comment:
The closing bracket below should be on this line.

Reply:
Done.